template/initPlayerLocal.sqf

```diff
@@ -0,0 +1,9 @@
+#include "script_component.hpp"
+
+// Disable CUP street lights based on lighting levels (bad performance script)
+CUP_stopLampCheck = true;
+
+[{alive player}, {
+    // Add action to change view distance and group names
+    call FUNC(playerActions);
+}, [], -1] call CBA_fnc_waitUntilAndExecute;
```

Comment:
I would remove the duplicate of this from onPlayerRespawn and use `CBA_fnc_addPlayerAction` instead of the `addAction` command